doc/netplan.md

```diff
@@ -860,6 +941,11 @@ Examples:
 :    netplan ID of the underlying device definition on which this VLAN gets
      created.
 
+## Properties for device type ``dummies:``
```

Comment:
What I mean by this is that it probably should be in a completely separate PR, since it's not directly related to tunnels.

Reply:
Well, yes, I agree it doesn't belong here. It just was slapped on top in hurry, because of how systemd-networkd refuses to up the bridge interface if it doesn't contain at least one other interface.  

Can you share your plans?